pkg/products/rhssouser/prometheusRules.go

```diff
@@ -72,6 +73,77 @@ func (r *Reconciler) newAlertsReconciler(logger l.Logger, installType string) re
 					},
 				},
 			},
+
+			//SLO-availability-alerts:
+			//https://sre.google/workbook/alerting-on-slos/
+			//https://promtools.dev/alerts/errors
+			{
+				AlertName: "user-sso-slo-availability-alerts",
+				Namespace: r.Config.GetOperatorNamespace(),
+				GroupName: "user-sso-slo-availability.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: fmt.Sprintf("%sUserSsoAvailability5mto1hErrorBudgetBurn", strings.ToUpper(installationName)),
```

Comment:
why prefix this with the installation type? Are there different installation types expected on the same cluster?

Reply:
Thank you very much for Review. Yes, it could be 2 installation types, as according to https://github.com/integr8ly/integreatly-operator  Readme.md:  "...An Integreatly Operator can be installed using two different flavours: managed or managed-api", where "managed" is for RHMI, and "managed-api" - for RHOAM.
According to another comment, I need try check another syntax, - 'haproxy_backend_http_responses_total{route=~"^keycloak.*", exported_namespace=~"{{ index .Params "namespace-prefix" }}.*sso$"}'. I'm sorry for problems, I will send update, if any change. Thanks a lot

Reply:
Federation of metrics -- updated and verified, working correctly,  same way as previous. Current/latest syntax: exported_namespace=~"{{ index .Params "namespace-prefix" }}.*sso$"
Thank you very much for Review.